src/domain/Loan.ts

```diff
@@ -16,16 +16,24 @@ export function updateLoans(pool: Pool, pileAddress: string): BigInt[] {
     let loans = pool.loans
     let loanId = loans[j]
 
+    let loan = Loan.load(loanId)
+    if (!loan) {
+      log.error('updateLoans: loan {} not found', [loanId])
+      return [new BigInt(0), new BigInt(0)]
+    }
+
+    // Ignore closed loans
+    if (loan.closed === 0) {
+      log.info('updateLoans: loan {} is closed. Will not proceed with update', [loanId])
+      return [new BigInt(0), new BigInt(0)]
```

Comment:
This will return a total debt of 0 for the entire pool if there's just one loan that's closed. I think this should instead be:
```suggestion
      continue
```